nuxt/components/form/Form.vue

```diff
@@ -12,13 +12,17 @@
       <div class="flex flex-col min-h-0 overflow-y-auto gap-6">
         <slot />
         <div class="flex flex-col items-center justify-between">
+          <div class="flex justify-center mb-4">
```

Comment:
Please use [`gap`](https://tailwindcss.com/docs/gap) on the parent element instead of `m(argin)`s on children.